src/ray/raylet/main.cc

```diff
@@ -38,6 +39,24 @@ static std::vector<std::string> parse_worker_command(std::string worker_command)
   return result;
 }
 
+ray::rpc::ClientTableData PopulateRayletInfo(
```

Comment:
Better to use a `RayletConfig` and `Builder` to instead of accessing pb like:
```C++
RayletConfig config = RayletConfigBuilder().SetClientID(random()).SetXXX().Build();
raylet = Raylet(config);
```

Reply:
> Better to use a `RayletConfig` and `Builder` to instead of accessing pb like:
> 
> ```c++
> RayletConfig config = RayletConfigBuilder().SetClientID(random()).SetXXX().Build();
> raylet = Raylet(config);
> ```
Both way it's ok.